src/lib/Libifl/auth.c

```diff
@@ -478,6 +488,8 @@ _invoke_pbs_iff(int psock, char *server_name, int server_port, char *ebuf, size_
 	/* for compatibility with 12.0 pbs_iff */
 	(void)snprintf(cmd[1], sizeof(cmd[1]) - 1, "%s -i %s %s %u %d %u", pbs_conf.iff_path, pbs_client_addr, server_name, server_port, psock, psock_port);
 #ifdef WIN32
+	if (pbs_conf.encrypt_method[0] != '\0')
+		SetEnvironmentVariable(PBS_CONF_ENCRYPT_METHOD, pbs_conf.encrypt_method);
```

Comment:
Can we check the return value of SetEnvironmentVariable and use GetLastError or the corresponding API in case of an error.